test/puppetlabs/trapperkeeper/services/authorization/authorization_service_test.clj

```diff
@@ -5,7 +5,7 @@
     [puppetlabs.ssl-utils.core :as ssl-utils]
     [puppetlabs.trapperkeeper.app :refer [get-service]]
     [puppetlabs.trapperkeeper.authorization.rules :as rules]
-    [puppetlabs.trapperkeeper.authorization.testutils :refer :all]
+    [puppetlabs.trapperkeeper.authorization.testutils :as testutils :refer :all]
```

Comment:
The file already had a :refer :all which infuriated me so I added an :as for the imports I used. I can audit the file and make it all relative and take the :all out.